embed.js

```diff
@@ -23,8 +23,10 @@ if (req.status == 404) {
         // URL: "https://www.madhappy.com/collections/accessories/products/classics-dad-hat?variant=39329359396975"
     })
         .then(function (response) {
+        if (response.data.reason != "redirects not enabled") { 
```

Comment:
why is this a string and not a boolean? Pass back a boolean in the response called enabled.